linux-tools/sqlite/sqlite.sh

```diff
@@ -29,9 +29,10 @@
 ## source the utility functions
 
 #cd `dirname $0`
-#LTPBIN=${LTPBIN%/shared}/sqlite
+LTPBIN="/usr/local/autotest_base/autotest-client-tests/linux-tools/shared"
 source $LTPBIN/tc_utils.source
```

Comment:
yes ,since will be exporting path in python wrapper file  before running tests,we should comment out this part .